src/python/mod.rs

```diff
@@ -144,6 +144,13 @@ pub fn py_sig_hash(
     Ok(bytes.into())
 }
 
+#[pyfunction(name = "wif_to_bytes")]
+pub fn py_wif_to_bytes(_py: Python, wif: &str) -> PyResult<PyObject>{
```

Comment:
As this parameter is used in the function, it should be renamed to py (remove the underscore)

Reply:
removed

Reply:
not removed .. changed to py